src/App.tsx

```diff
@@ -1,109 +1,37 @@
 import React from 'react';
 
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import { User } from './types/User';
+import { Comments } from './types/Comments';
+import { Posts } from './types/Posts';
+
+import { PostList } from './components/PostList';
+
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+function getUsersById(userId:number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getPostsById(postId:number):Comments[] {
+  return commentsFromServer.filter(comment => comment.postId === postId) || [];
```

Comment:
```suggestion
  return commentsFromServer.filter(comment => comment.postId === postId);
```
filter returns an empty array  anyway